content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/csi-node-v1.md

```diff
@@ -6,7 +6,8 @@ api_metadata:
 content_type: "api_reference"
 description: "CSINode 包含节点上安装的所有 CSI 驱动有关的信息。"
 title: "CSINode"
-weight: 9
+weight: 4
+auto_generated: true
```

Comment:
In translated version, we intended to remove the `auto_generated` field.

Reply:
ok, removed